sql.txt

```diff
@@ -18,7 +18,19 @@ WHERE ProductID = 51 AND Quantity > 10;
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES ('ABC', 1, 1, '2 bottles', 69);
+
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES ('XYZ', 1, 1, '3 boxes', 5);
+
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES ('CTD', 1, 1, '1 package', 13);
 
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
+UPDATE TOP (2) Products
+SET Price = Price + 1.5;
 
```

Comment:
I think you were having technical issues using top and update in #5 earlier. Another way of writing it could have been:
UPDATE Products
SET Price = Price +1.5
Limit 2;